src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+
+As far as Aristotle was concerned then, an element cannot be divided into bodies of a different form, such that if one were to divide "air", one would not receive "fire" or "earth". Whether this is still valid for Hegel is not clear here, but what is clear is that logic principally understood as element persists throughout its many divisions or transformations and is therefore _the_ unity. 
+
+However, what might contrast to the thinking of element as a kind of "building block" is the fact that the logical developments of the concept are non-regressive. That is to say, the progression of thought cannot be reverted back to its constituents. As Hegel confirms:
+
+> For since the division is ... the _parting_ or the _judgment_ of the concept &ndash; is the positing of the determination which is already immanent in it and therefore the positing of its distinction &ndash; this positing must not be understood as resolving that concrete unity back into its determinations, as if these were to exist on their own, for this would be here a vacuous return to the previous standpoint, to the opposition of consciousness. But this opposition has vanished; the unity remains the element, and the distinctions of the division and of the development in general no longer transgress that unity (Hegel, 2010, 39/21.45). 
+
+Positing&mdash;here meaning putting something as explicit&mdash;develops the content further and reveals new details about it. This further account cannot be said to be part of the constituents as they were but must be part of them as they develop. In the context of thought and consciousness, the latter is oriented around an subject-object opposition whereas in thought, by contrast, this opposition has _vanished_, leaving logic as the sole element of which any further distinctions and classifications form moments. 
+
+### Logic from Kant to Hegel (Niklas)
+
+Hegel compares his logic and its constituents to Kant's and notes some differences. First, without controversy, Kant makes a distinction between general logic and transcendental logic, the latter being the logic concerning experience or cognition (or how objects appear) whilst the former being the logic that concerns logic as such. For Kant, while transcendental logic may be erroneous or differ depending on the kind of creature that is in question (human being, God, alien, etc.), general logic can never be in error. Said otherwise, my cognitive apparatus generates a particular kind of experience of the world that is peculiar to me _as a human being_ and outside of which I cannot make any truthful claims about the world and its object; but these experiences or claims must accord with the rules of general logic, such that I cannot have an experience that would ever violate general logic. However, general logic alone is insufficient to provide any positive claims about the world and its objects. 
```

Comment:
this paragraph ends a bit abruptly. It would be fine if it ended with "violate general logic", but the fact that it ends with a sentence that begins with "However", gives the impression that there's more to follow.

Reply:
Rephrased.